### PR TITLE
Actions: Add method overloads for `PercentType` to Audio & Voice

### LINKED ISSUE #3331: [actions] Accept float for Audio and Voice actions in addition to PercentType
Coming from https://github.com/openhab/openhab-js/issues/98#issuecomment-1367278509, I wonder if the Audio and Voice actions could be extended to also accept float for the volume in addition to the PercentType.

This addition would be useful to all scripting addons, because script engines can convert e.g. JS numbers to float, but can’t convert a JS number to an openHAB PercentType. In theory, it would be possible to wrap those actions inside the scripting libraries, but then every script language would need to wrap. 

IMO it would be instead better to enable core to accept float as well for all Audio and Voice actions. For some it is already implemented:

https://github.com/openhab/openhab-core/blob/673827733ae870f13937e58ccd87ab04c02673ec/bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java#L105-L111

@J-N-K What’s your opinion here? I can implement this myself, but I’d like to get some feedback here so that I don’t start working on something that won’t get accepted.

### PR DESCRIPTION
**Pull request by florian-h05**
Closes #3331.
Related to https://github.com/openhab/openhab-js/issues/98.

This PR adds method overloads for Audio and Voice actions to accept float as volume in addition to `PercentType`.
This is especially useful when accessing these actions from the JSR223 languages because you do not need to import and construct a `PercentType` then.

I also aligned the params for the methods to increase readability of code and corrected the mathematical notation of intervals in strings (correct is `;` instead of `,`).

### CODE REVIEW

File: bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java [hunk 1]
```diff
@@ -47,17 +47,23 @@ public static void playSound(@ParamDoc(name = "filename", text = "the filename w
 
     @ActionDoc(text = "plays a sound with the given volume from the sounds folder to the default sink")
     public static void playSound(@ParamDoc(name = "filename", text = "the filename with extension") String filename,
-            @ParamDoc(name = "volume", text = "the volume to be used") PercentType volume) {
+                                 @ParamDoc(name = "volume", text = "the volume to be used") PercentType volume) {
```
Comment (J-N-K) [inline bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java:50]: What happened to the formatting here?

File: bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java [hunk 2]
```diff
@@ -96,18 +109,15 @@ public static synchronized void playStream(@ParamDoc(name = "sink", text = "the
...
     }
 
     @ActionDoc(text = "sets the master volume")
     public static void setMasterVolume(
-            @ParamDoc(name = "volume", text = "volume in the range [0,1]") final float volume) throws IOException {
-        if (volume < 0 || volume > 1) {
-            throw new IllegalArgumentException("Volume value must be in the range [0,1]!");
-        }
-        setMasterVolume(new PercentType(new BigDecimal(volume * 100f)));
+            @ParamDoc(name = "volume", text = "volume in the range [0;1]") float volume) throws IOException {
```
Comment (J-N-K) [inline bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java:119]: Can you re-insert the `final` for consistency here?

**Review by J-N-K — CHANGES_REQUESTED** [review]
In general LGTM, the comment regarding the formatting applies to different locations.

File: bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java (hunk 1 above)
Comment (florian-h05) [reply bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java:50]: I aligned all param docs for better readability, I can revert if you want.

Comment (J-N-K) [reply bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java:50]: no, fine then.

File: bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java (hunk 2 above)
Comment (florian-h05) [reply bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java]: Done.

**Review by J-N-K — APPROVED** [review]
Thanks!

**Pull request merged**